pkg/adapter/nimbus-kyverno/processor/kpbuilder.go

```diff
@@ -239,75 +351,313 @@ func cocoRuntimeAddition(np *v1alpha1.NimbusPolicy) ([]kyvernov1.Policy, error)
 	return kps, multierr.Combine(errs...)
 }
 
-func escapeToHost(np *v1alpha1.NimbusPolicy, rule v1alpha1.Rule) kyvernov1.Policy {
+func virtualPatch(np *v1alpha1.NimbusPolicy) ([]kyvernov1.Policy, error) {
+	rule := np.Spec.NimbusRules[0].Rule
+	requiredCVES := rule.Params["cve_list"]
+	//  tbd
+	// schedule := rule.Params["schedule"][0]
```

Comment:
Why is `tbd` here? What's left?

Reply:
Sorry, left the commented code as it is, removing it now.